### PR TITLE
Add: Project specification documentation

### LINKED ISSUE #1: Add Project Specification
Aspek project specification:
- Tekologi yang akan digunakan
- Project Goals
- Project deliverables
- Project scope

### PR DESCRIPTION
**Pull request by afifabroory**
Resolve #1

### CODE REVIEW

File: docs/Specifications.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- Kontent deskripsi/catatan tugas dan project harus mendukung format penulisan Markdown.
+- Untuk meringankan penggunaan [back end](#technology), deskripsi tugas dan project tidak disimpan pada back end. Melainkan disimpan di repo khusus GitHub organization ini dalam bentuk markdown. Sehingga pada back end hanya menyimpan sebgian data yang rinciannya sebagai berikut:
+    - Kode tugas dan project
+    - Judul tugas dan project
+    - Metadata tugas dan project
+    - Status tugas masing-masing anggota
+    - Token registrasi
+    - _TBD._ (To Be Determined)
+
+# User Story (Non Standard)
```
Comment (afifabroory) [inline docs/Specifications.md:27]: Why non standard? But, ok.

**Review by afifabroory — COMMENTED** [review]
Need review.

File: docs/Specifications.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Project Goals
```
Comment (mdaffailhami) [inline docs/Specifications.md:1]: Ini nge-review :v

Comment (mdaffailhami) [inline docs/Specifications.md:1]: ## Ini ngomen :v

Comment (bhaktiramadhani) [inline docs/Specifications.md:1]: test komen

**Review by bhaktiramadhani — COMMENTED** [review]
test

**Pull request merged**